lab_5_scrapper/scrapper.py

```diff
@@ -100,6 +138,10 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    return requests.get(url,
```

Comment:
Make sure you use all the provided configuration settings, including encoding etc.